kafka_enforcer_common/src/main/java/com/tesla/data/enforcer/BaseCommand.java

```diff
@@ -81,6 +82,10 @@ public BaseCommand(Map<String, Object> cmdConfig, String cluster) {
     return MAPPER.convertValue(cmdConfig().get("kafka"), MAP_TYPE);
   }
 
+  public String getZookeeperConnection() {
```

Comment:
I imagine there are a number of ZK configuration properties besides just the connection string that we might want - timeouts look to be one of them. In generally, this property is the `zookeeper.connect` string. 

So a parent `zookeeper` section in the YAML would also be appropriate and we could have a similar model to the kafkaConfig block above

Reply:
makes sense. Removed this bit for now as the refactored code doesn't explicitly deal with Zooekeeper. Will keep this in mind for upcoming PRs though